report-service/src/main/java/gov/cdc/dataingestion/kafka/integration/service/KafkaConsumerService.java

```diff
@@ -45,12 +47,17 @@ public class KafkaConsumerService {
 
     @Value("${kafka.raw.topic}")
     private String rawTopic = "";
+
+    @Value("${kafka.elr-validated-dlt.topic}")
+    private String validatedElrDltTopic = "";
```

Comment:
perhaps change the name of this topic to something like duplicated-elr since this is not actually created by dead letter topic process